### DISCUSSION TITLE
How to uninstall code-server ?

### DISCUSSION THREAD

**Discussion by enniosousa** [coder/code-server#1989] [Q&A] [answered]
How to uninstall code-server ?

- Remote OS: Ubuntu 18.04.2 LTS
- Remote Architecture:
- `code-server --version`: v3.3.1
- VS Code: v1.45.1

**Comment by aptalca** [comment]
How did you install it?

**Comment by rainmanjam** [comment]
I used the installation script. 

`curl -fsSL https://code-server.dev/install.sh | sh`

How do you uninstall code-server?

**Comment by bpmct** [comment, marked as answer]
> You can just `rm -r` the `code-server` directory. It depends on where you installed it, but if you used the install script by default it'll be in `~/.local/lib/code-server-<version>` and there will also be a binary symlink in `~/.local/bin/code-server` you can remove.
> 
> If you want to delete settings and data as well you can remove `~/.local/share/code-server` and `~/.config/code-server`.
Source: https://github.com/cdr/code-server/issues/1946#issuecomment-670575158

**Comment by 0xSeanll** [comment]
For people using install script, the directory has changed to `/usr/lib/code-server`.

**Comment by ghost** [comment]
```
rm -rf /usr/lib/code-server/
rm -rf $HOME/.config/code-server/
rm -rf $HOME/.cache/code-server
rm -f $HOME/.local/share/code-server
rm -f $HOME/.local/bin/code-server
rm -f $HOME/.local/lib/code-server-*
```

**Reply by lg-nineoneninethreenine** [reply]
> ```
> rm -rf /usr/lib/code-server/
> rm -rf $HOME/.config/code-server/
> rm -rf $HOME/.cache/code-server
> rm -f $HOME/.local/share/code-server
> rm -f $HOME/.local/bin/code-server
> rm -f $HOME/.local/lib/code-server-*
> ```

This could be better, so that users don't nee to run additional commands over each other.

**Reply by ghost** [reply]
Thanks, fixed.

**Reply by jmkim** [reply]
`rm -rf $HOME/.local/share/code-server/` instead of `rm -f $HOME/.local/share/code-server`. It is a *directory* for user preferences and extensions.

**Reply by daanta-real** [reply]
I love you.

**Comment by erbanku** [comment]
[Uninstall code-server (from the official document)](https://coder.com/docs/code-server/latest/install#uninstall)

**Reply by OmarLaham** [reply]
Great one! Thanks =)

**Comment by rrkroms** [comment]
You can just remove( rm -r ) the code-server directory. It depends on where you installed it, but if you used the install script by default it'll be in ~/.local/lib/code-server-<version> and there will also be a binary symlink in ~/.local/bin/code-server you can remove.

If you want to delete settings and data as well you can remove ~/.local/share/code-server and ~/.config/code-server , ~.cache/code-server.

**Comment by marcosadsj** [comment]
[WSL 2]

i just deleted the bin folder using command `sudo rm -R /home/mysusername/.vscode-server/bin/` and then I ran `code .` 

As a result VSCode reinstalled along with all my data and installed extensions: 

Installing VS Code Server for x64 (2b35e1e6d88f1ce073683991d1eff5284a32690f)
Downloading: 100%
Unpacking: 100%
Unpacked 1522 files and folders to /home/mysusername/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f.

**Reply by tteck** [reply]
try `dpkg -r code-server`

**Reply by marcosadsj** [reply]
The solution that I wrote above worked for me. But anyway, thank you @tteck
